src/test/java/com/getindata/connectors/http/internal/table/lookup/LookupQueryInfoTest.java

```diff
@@ -35,10 +35,54 @@ public void testEmptyLookupQueryInfo() {
     public void testBodyBasedUrlQueryParams() {
         Map<String, String> bodyBasedUrlQueryParameters = Map.of("key1", "value1");
 
-        lookupQueryInfo = new LookupQueryInfo(null, bodyBasedUrlQueryParameters);
+        lookupQueryInfo = new LookupQueryInfo(null, bodyBasedUrlQueryParameters, null);
 
         assertThat(lookupQueryInfo.hasLookupQuery()).isFalse();
         assertThat(lookupQueryInfo.hasBodyBasedUrlQueryParameters()).isTrue();
         assertThat(lookupQueryInfo.getBodyBasedUrlQueryParameters()).isEqualTo("key1=value1");
     }
+
+    @Test
+    public void testPathBasedUrlParams1() {
```

Comment:
nit: Could you provide more meaningful test names? So the difference between `testPathBasedUrlParams1` and `testPathBasedUrlParams2` is clear just by reading test name.